src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -950,6 +950,10 @@ registerThemingParticipant((theme: ITheme, collector: ICssStyleCollector) => {
 			.monaco-workbench > .part.editor > .content > .one-editor-silo > .container > .title.active .tabs-container > .tab:hover  {
 				background: ${tabHoverBackground} !important;
 			}
+
+			.monaco-shell:not(.hc-black) .monaco-workbench > .part.editor > .content > .one-editor-silo > .container > .title.active .tabs-container >  .tab.sizing-shrink:hover > .tab-label::after {
+				background: linear-gradient(to left, ${theme.getColor(TAB_HOVER_BACKGROUND)}, transparent);
```

Comment:
@Dari-K in many places here you seem to unnecessarily call `theme.getColor(TAB_HOVER_BACKGROUND)` even though the color was already resolved from the step before.